src/mongoose_metrics_definitions.hrl

```diff
@@ -77,6 +77,9 @@
            eval, ?EX_EVAL_SINGLE_VALUE}},
          {nodeUpTime,
           {function, mongoose_metrics, get_up_time, [],
+           tagged, [value]}},
+         {tcpCountConnections,
```

Comment:
It would be more accurate to call this metric `tcpPortsUsed`, as it includes ports used for listening as well.